ocs_ci/utility/utils.py

```diff
@@ -1796,3 +1797,31 @@ def skipif_ocs_version(expressions):
         skip_this = skip_this and eval(comparision_str)
     # skip_this will be either True or False after eval
     return skip_this
+
+
+def get_ocs_version_from_tag(tag):
+    """
+    Parse major.minor version from OCS image tag.
+
+    Args:
+        tag (str): version tag from the image
+
+    Returns
+        str: Version in x.y format
+
+    Raises:
+        ValueError: In case tag which we cannot parse passed.
+
+    """
+    try:
+        version = Version(tag.split(':')[1].lstrip("latest-"))
```

Comment:
Above will give ValueError if we pass quay.io/rhceph-dev/ocs-olm-operator:latest-4.2

Reply:
``` >>> tag = "quay.io/rhceph-dev/ocs-olm-operator:latest-4.2"
>>> Version(tag.split(':')[1].lstrip("latest-"))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/vavuthu/VJ/Gluster/automation/commits/MASTER/venv1/lib64/python3.7/site-packages/semantic_version/base.py", line 105, in __init__
    major, minor, patch, prerelease, build = self.parse(version_string, partial)
  File "/home/vavuthu/VJ/Gluster/automation/commits/MASTER/venv1/lib64/python3.7/site-packages/semantic_version/base.py", line 311, in parse
    raise ValueError('Invalid version string: %r' % version_string)
ValueError: Invalid version string: '4.2'
>>> 
>>> 
>>> Version.coerce(tag.split(':')[1].lstrip("latest-"))
Version('4.2.0')
>>> 
```

Reply:
If you treat `latest-` in the exception below, you can probably just skip lstrip here, right?
```suggestion
        version = Version(tag.split(':')[1])
```